src/Feature/PageContent/client/BannerGrid/index.tsx

```diff
@@ -30,7 +29,7 @@ import {
 
 import './styles.scss';
 
-class HeroHomeControl extends Jss.SafePureComponent<HeroHomeControlProps, HeroHomeControlState> {
```

Comment:
Banner grid is a rendering with placeholders in it.

Reply:
Yes, this change will added in another task for this story